main/getting-started/start-a-project.md

```diff
@@ -35,70 +32,36 @@ This presentation includes starting a project, but note an outdated detail:
 Use the [Agoric CLI](/guides/agoric-cli/commands.md) to fetch from a Dapp template
 and put it in a `demo` directory _not located in your `agoric-sdk` clone_:
 
-
 ```sh
 # Terminal 1
-# Don't use your agoric-sdk clone as the parent of the demo directory.
-# It doesn't have to be your home directory; any other directory will do.
+# Don't use your agoric-sdk as the parent of the demo directory.
 cd $HOME
-agoric init demo
+agoric init demo # use `agoric init $DIRNAME` with any name you like
+cd demo
+agoric install
+agoric start --verbose # `agoric start --reset` to start over
 ```
 
-The name `demo` is an arbitrary suggestion; in general,
-use `agoric init $DIRNAME` with any name you like.
+The name `demo` is an arbitrary suggestion. The default template is the [Fungible Faucet Dapp](https://github.com/Agoric/dapp-fungible-faucet).
 
-The default template is the [Fungible Faucet Dapp](https://github.com/Agoric/dapp-fungible-faucet).
 Learn more about the [available dapp templates](/dapps/dapp-templates.md).
 
-### Install the Agoric SDK in the Dapp
-
-```sh
-# Terminal 1
-cd demo
-agoric install
-```
-
 It may take a minute or so to install all the dependencies.
 
 ::: tip Mac Dev Tools
 On a Mac, you must first install
 [Xcode](https://apps.apple.com/us/app/xcode/id497799835)
 :::
 
-## Start the Agoric Solo Client and Simulated Blockchain
```

Comment:
Likewise, this heading seems important for building a mental model. It seems like a PR that deletes this heading is incomplete without adding some explanation elsewhere.

Reply:
I added a short section below the sub heading "Initialize `demo`..." that summarizes these two lost headings.

Reply:
ah. good.